downloads/swan-lake-release-notes/swan-lake-2201.1.0/RELEASE_NOTE.md

```diff
@@ -0,0 +1,689 @@
+int[*][*] x4 = [[1, 2], [1, 2]]; // Not supported. Only the first dimension can be inferred.
+```
+
+### Bug fixes
+
+- Fixed an invalid sub-typing relationship between `table` and `anydata` 
+
+    ```ballerina
+    public function main() {
+        table<map<any>> tany = table [{"a": 2}];
+        anydata ad = tany; // Results in a compilation error now.
+    }
+    ```
+
+- Fixed an issue that caused a union containing the `null` literal allowing `"null"` as a valid value
+
+    ```ballerina
+    type Foo boolean|null;
+
+    public function main() {
+        Foo a = "null"; // Results in a compilation error now.
+        "string"|null b = "null"; // Results in a compilation error now.
+    }
+    ```
+
+- Fixed an issue that caused the value of enum members defined with quoted identifiers to include the quote
+
+    ```ballerina
+    import ballerina/io;
+
+    public enum Status {
+        'new,
+        old
+    }
+
+    public function main() {
+        io:println('new); // Previously printed `'new`, now prints `new`.
+    }
+    ```
+
+- Fixed a bug that resulted in a compilation error not being logged for an extra comma in a mapping match pattern
+
+    ```ballerina
+    type MyRecord record {
+        int field1;
+        int field2;
+    };
+
+    function fn(MyRecord r1) {
+        match r1 {
+            {field1: 0,} => { // A syntax error is now given for the comma.
+
+            }
+        }
+    }
+    ```
+
+- Fixed qualified identifiers not being allowed in error match patterns 
+
+    ```ballerina
+    function fn(error e) {
+        match e {
+            error(errors:MESSAGE) => { // Match pattern is now allowed.
+
+            }
+        }
+    }
+    ```
+
+- Fixed the inherent type of a list constructed using a list constructor with `any` as the contextually-expected type to be `(any|error)[]` instead of `any[]`
+
+    ```ballerina
+    public function main() {
+        any x = [1, 3, 4];
+        if x is (any|error)[] {
+            x.push(error("invalid!")); // Allowed now. Previously, failed at runtime.
+        }
+    }
+    ```
+
+- Fixed a bug that allowed additive expressions with operands of types that are union types of different basic types 
+
+    ```ballerina
+    public function main() {
+        int|float a = 4;
+        int|float b = 4.5;
+
+        int _ = a + b; // Now, this results in a compilation error.
+    }
+    ```
+
+To view bug fixes, see the [GitHub milestone for Swan Lake 2201.1.0](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+is%3Aclosed+label%3AType%2FBug+label%3ATeam%2FCompilerFE+milestone%3A%22Ballerina+2201.1.0%22).
+
+## Runtime updates
+
+### Improvements
+
+#### Support to provide values for configurable variables through TOML in-line tables
+
+The configurable feature is improved to support TOML in-line tables through the TOML syntax.
+The values for configurable variables of types `map` and `record` can now beprovided using TOML in-line tables.
```

Comment:
```suggestion
The values for configurable variables of types `map` and `record` can now be provided using TOML in-line tables.
```

Reply:
Sorry, this was already fixed and I have missed pushing the commit https://github.com/ballerina-platform/ballerina-dev-website/pull/4402/commits/5c848ddf159295d59ad8a1b96eebea73d7252d12.